### ISSUE TITLE
[Boss/Spell] Professor Putricide variables

### ISSUE THREAD

**Issue by Oganku** [WoWManiaUK/Redemption#4385]
**Links:** Related to #4351
https://www.wowhead.com/spell=70353/gas-variable
https://www.wowhead.com/spell=70352/ooze-variable 

**What is Happening:** The [Ooze ](https://www.wowhead.com/spell=70352/ooze-variable ) and [Gas ](https://www.wowhead.com/spell=70353/gas-variable
) variables cast upon the raid during the transition against Professor Putricide on 25 heroic never gets dispelled even after all oozes (usually 2) are killed and none are left alive.

**What Should happen:** When the transition oozes are killed, the variables placed on the group should be removed. This used to work just fine before, and I feel #4351 didn't 'fix' anything and instead it broke it now. The variable used to be removed only once both oozes died, and getting 3 oozes in a transition on 25hc is pretty much a wipe regardless.

**Comment by Keader** [comment]
Check https://github.com/WoWManiaUK/Redemption/issues/4351
Will be fixed tomorrow <3

**Comment by Oganku** [comment]
Oh, so #4200 is the issue and #4351 the fix? My bad for misunderstanding then

**Comment by Keader** [comment]
yes, i put a new issue, to appear in new changelog
